.factory/automation.yml

```diff
@@ -59,12 +59,12 @@ build:
 #      command: |
 #        bazel run @vaticle_dependencies//tool/bazelinstall:remote_cache_setup.sh
 #        bazel test //test/behaviour/connection/... --test_output=streamed
-#    test-behaviour-concept:
-#      image: vaticle-ubuntu-22.04
-#      dependencies: [build]
-#      command: |
-#        bazel run @vaticle_dependencies//tool/bazelinstall:remote_cache_setup.sh
-#        bazel test //test/behaviour/concept/... --test_output=streamed
+    test-behaviour-concept:
+      image: vaticle-ubuntu-22.04
+      dependencies: [build]
+      command: |
+        bazel run @vaticle_dependencies//tool/bazelinstall:remote_cache_setup.sh
+        bazel test //tests/behaviour/concept/... --test_output=streamed
```

Comment:
I don't want to worry about unit tests now, but I don't want to continue fixing bdds after random commits from us, so I believe it's worth adding concept bdds (as they are finished and just wait for some minor todos based on the future discussions) into CI